package.json

```diff
@@ -11,31 +11,36 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~11.1.1",
-    "@angular/common": "~11.1.1",
-    "@angular/compiler": "~11.1.1",
-    "@angular/core": "~11.1.1",
-    "@angular/forms": "~11.1.1",
-    "@angular/localize": "~11.1.1",
-    "@angular/platform-browser": "~11.1.1",
-    "@angular/platform-browser-dynamic": "~11.1.1",
-    "@angular/router": "~11.1.1",
+    "@angular/animations": "^11.2.13",
+    "@angular/common": "^11.2.13",
+    "@angular/compiler": "^11.2.13",
+    "@angular/core": "^11.2.13",
+    "@angular/forms": "^11.2.13",
+    "@angular/localize": "^11.2.13",
+    "@angular/platform-browser": "^11.2.13",
+    "@angular/platform-browser-dynamic": "^11.2.13",
+    "@angular/router": "^11.2.13",
```

Comment:
was this necessary to change or should we just use the dev version of this file?
Same question for package-lock.json, angular.json and .vscode/launch.json.